### ISSUE TITLE
Is it possible to add new optimization problems?

### ISSUE THREAD

**Issue by Alaya-in-Matrix** [uber/bayesmark#17]
Instead of using the sklearn models, is it possible to add new optimization problems in bayesmark?

**Comment by wen-zheng** [comment]
of course, bayesmark is a benchmark for different optimization algorithms, and it provides a set of uniform  interfaces to access these algorithms.
